example_apps/interal_sdk_inference_orchestrator_example_app/README.md

```diff
@@ -0,0 +1,76 @@
+# Internal SDK Inference Orchestrator example App
+
+## Requirements
+- Python 3.9+
```

Comment:
since our sdk use 3.10, we need to update this

Reply:
Done